# EnkeLab-Gallusgallus-Data
This repository contains the necessary instructions and code in order to perform a differential gene expression analysis on Gallus gallus RNA-Seq data

This repository contains code used for quality control and data analysis presented in: 

> Fill in Later

----

## Data availability

Fill in later

## Software used

| Software | Version | URL | 
| --- | --- | --- |
| FastQC | 0.11.5 | http://www.bioinformatics.babraham.ac.uk/projects/fastqc/ |
| Trimmomatic | 0.36 | http://www.usadellab.org/cms/?page=trimmomatic  |
| Kallisto | 0.42.3 | https://pachterlab.github.io/kallisto/ |
| Sleuth |  | https://pachterlab.github.io/sleuth/ |

## Data analysis walkthroughs & code

Walkthroughs and code used for all of the quality assessment and data analysis steps are available in each of the below links.

1. [Quality assessment with FastQC](https://github.com/ScottSchumacker/EnkeLab-Gallusgallus-Data/blob/master/Walkthroughs-Code/FastQC)
1. [Sequence trimming with Trimmomatic](https://cyverse-trimmomatic-quickstart.readthedocs-hosted.com/en/latest/)
1. [Kallisto indexing and quanitification](https://learning.cyverse.org/projects/kallisto_tutorial/en/latest/)
1. [Sleuth Part 1 Data Quality Analysis](https://learning.cyverse.org/projects/kallisto_tutorial/en/latest/step4.html)
1. [Sleuth part 2 Differential Expression Analysis]()
